app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,77 @@
+class FoodsController < ApplicationController
+  before_action :set_food, only: %i[show edit update destroy]
+
+  # GET /foods or /foods.json
+  def index
+    @foods = Food.all
```

Comment:
- [ ] Nice work making the the index page for food match the given wireframe :clap: However it would be nice to display only the foods that belong to the logged-in user to follow the project requirement.
![Screenshot from 2024-01-26 17-43-45](https://github.com/Surafels/recipe-app/assets/91142432/9308ae6f-ae27-4949-855a-c67e95eb16bf)